src/sentry/static/sentry/app/views/acceptOrganizationInvite.tsx

```diff
@@ -95,47 +96,59 @@ class AcceptOrganizationInvite extends AsyncView<Props, State> {
 
     return (
       <React.Fragment>
-        <p>
-          {t(
-            `To continue, you must either login to an existing Sentry account,
-             or create a new account.`
-          )}
-        </p>
+        {!inviteDetails.requireSso && (
+          <p data-test-id="action-info-general">
+            {t(
+              `To continue, you must either create a new account, or login to an
+              existing Sentry account.`
+            )}
+          </p>
+        )}
         {inviteDetails.needsSso && (
-          <p data-test-id="suggests-sso">
+          <p data-test-id="action-info-sso">
             {tct(
-              `Note that [orgSlug] has enabled Single-Sign-On (SSO) using
+              `Note that [orgSlug] has [isRequired] Single Sign-On (SSO) using
                [authProvider]. You may create an account by authenticating with
-               the organizations SSO provider.`,
+               the organization's SSO provider.`,
               {
-                orgSlug: inviteDetails.orgSlug,
+                orgSlug: <strong>{inviteDetails.orgSlug}</strong>,
                 authProvider: inviteDetails.ssoProvider,
+                isRequired: inviteDetails.requireSso ? 'required' : 'enabled',
```

Comment:
should this be also translated or it's fine only pass them to the translation? in this case, I usually use 2 tct